Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -5,13 +5,25 @@ extension Style {
 
     /// This function creates an expression that will localize the `textField` property of a `SymbolLayer`
     /// - Parameter locale: A `SupportedLanguage` based `Locale`
-    internal func localizeLabels(into locale: Locale) throws {
+    public func localizeLabels(into locale: Locale, for layerIds: [String]? = nil) throws {
+
+        /// Do nothing if we do not support the locale
+        if !supportedLocales.contains(locale) {
+            return
```

Comment:
Would it be better to throw?

Also, `///` is for documentation comments. Normal comments can use `//`